_component_code/button-bars.md

```diff
@@ -2,7 +2,7 @@
 title: Button Bars
 layout: code
 category: Components Code
-code: components/menus
+code: components/menu
```

Comment:
Should this point to `menu` or `button`?

Reply:
See the conversation on the design page below on that question. I think it's a designer question.